### PR TITLE
Mass Dialog Updates #1

### PR DESCRIPTION
**Pull request by fedprod**
Over the next few days I'll go over all dialogues, to check for inconsistencies and updates.

### CODE REVIEW

File: Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/OBJ_DUDE.MSG [hunk 1]
```diff
@@ -12,63 +12,231 @@
...
 # HOTEL.MAP
 # The Necropolis Hotel Tell-Me-Abouts
+#
 {3400}{}{Meister}
-{3401}{}{Steuerchip}
-{3402}{}{Necropolis}
-{3403}{}{Set}
+{3401}{}{Steuerchip|Wasserchip|Chip}
+{3402}{}{Necropolis|Nekropolis|City of the Dead|City of Dead}
+{3403}{}{Set|Seth}
```
Comment (wipe2238) [inline Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/OBJ_DUDE.MSG]: `Set` and `Seth` are two different NPCs

File: Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/OBJ_DUDE.MSG [hunk 2]
```diff
@@ -12,63 +12,231 @@
...
 
 # WATRSHD.MAP
 # The Necropolis Watershed Tell-Me-Abouts
 {4000}{}{Meister}
-{4001}{}{Steuerchip}
-{4002}{}{Necropolis}
-{4003}{}{Set}
+{4001}{}{Steuerchip|Wasserchip|Chip}
+{4002}{}{Necropolis|Nekropolis|City of the Dead|City of Dead}
+{4003}{}{Set|Seth}
```
Comment (wipe2238) [inline Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/OBJ_DUDE.MSG]: again

File: Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/RAZLO.MSG [hunk 3]
```diff
@@ -109,21 +109,32 @@
...
+{1001}{}{Garl|Death Hand}
+{1002}{}{Junktown|Junk Town}
+{1003}{}{Shady Sands|Shady|Sands}
 {1004}{}{Hub}
-{1005}{}{Wasser}
+{1005}{}{Wasser|Steuerchip|Wasserchip|Chip}
 {1006}{}{Tandi}
 {1007}{}{Aradesh}
-{1008}{}{Seth}
+{1008}{}{Seth|Set}
```
Comment (wipe2238) [inline Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/RAZLO.MSG]: and again

File: Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/SET.MSG [hunk 4]
```diff
@@ -222,29 +222,22 @@
...
-{1019}{}{Children of the Cathedral}
-{1020}{}{Stelle}
-{1021}{}{Wasser}
+{1007}{}{Brotherhood of Steel|Brotherhood|Steel|BOS}
+{1008}{}{Wasserstelle|Wasserversorgung|Wasser|Stelle}
+{1009}{}{Cathedral|Children|Children of the Cathedral|CoC}
+{1010}{}{Steuerchip|Chip|Wasserchip}
+{1011}{}{Halle|Halle der Toten|Totenhalle|Halle des Todes}
+{1012}{}{Garret}
+{1013}{}{Set|Seth}
```
Comment (wipe2238) [inline Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/SET.MSG]: and again

**Comment by wipe2238** [comment]
There is extra space added at the start of some lines :/

File: Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/SET.MSG (hunk 4 above)
Comment (Lexx2k) [reply Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/SET.MSG]: This is on purpose. I've added them because they are easy to misstype.

Comment (wipe2238) [reply Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/SET.MSG]: You criminal dev :(

**Comment by Lexx2k** [comment]
> 
> 
> There is extra space added at the start of some lines :/

What lines? Sometimes player responses are marked with a tab in the beginning. This, also, is on purpose to make them easier to recognize when manually editing dialog files.

File: Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/OBJ_DUDE.MSG (hunk 1 above)
Comment (fedprod) [reply Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/OBJ_DUDE.MSG]: I know that, but people often missnaming them in my experience.

**Comment by fedprod** [comment]
What Lexx said. Also I've learned that that is the used indention for player responses in sfall's message editor, although I didn't use it throughout all files^^"

File: Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/SET.MSG (hunk 4 above)
Comment (fedprod) [reply Fallout2/Fallout1in2/mods/fo1_german/text/german/dialog/SET.MSG]: Or lazy/misremembering people... I can't tell how many times I've called Set Seth in my mind (although I would write them correctly and wonder "wait, why did I call him Set(h)?"
We're all getting older :^

**Pull request merged**